python_course_1604/class_01/exercize_03_capture_exception.py

```diff
@@ -23,3 +24,11 @@ def capute_exception():
     If any else exception is raised, then it must let it pass on.
     Please use try: except: else: construct.
     """
+    try:
+        failing_function()
+    except RuntimeError:
+        return False
+    except:
```

Comment:
This is not necessary: it is doing nothing. You can remove it as non captured exceptions are already raised.